src/ObjectMakr.ts

```diff
@@ -73,7 +72,9 @@ export class ObjectMakr implements IObjectMakr {
             this.processProperties(this.properties);
         }
 
-        this.processFunctions(this.inheritance, Object, "Object");
+        if(!this.functions){
```

Comment:
What happens if the user only passes in *some* of the required functions, and wants the rest to be auto-generated as normal?